### ISSUE TITLE
Janeway Front End Accessibility Issue Collection

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4259]

**Comment by StephDriver** [comment]
To further improve on WCAG 2.4.5 Multiple Ways after #4231
    - Improve Search
    - Introduce front end breadcrumbs

**Comment by StephDriver** [comment]
see away day discussion BirkbeckCTP/bau#80 on articles.

**Comment by StephDriver** [comment]
Review all breadcrumbs in the back office, and in particular consider the pages at the top level (i.e. any breadcrumb would only displaying their own page name, no links) like the Log in page and whether there should be any breadcrumb there.
